Formula/global.rb

```diff
@@ -23,12 +24,14 @@ class Global < Formula
   end
 
   option "with-ctags", "Enable Exuberant Ctags as a plug-in parser"
+  option "with-universal-ctags", "Enable Universal Ctags as a plug-in parser"
   option "with-pygments", "Enable Pygments as a plug-in parser (should enable exuberent-ctags too)"
   option "with-sqlite3", "Use SQLite3 API instead of BSD/DB API for making tag files"
 
   deprecated_option "with-exuberant-ctags" => "with-ctags"
 
   depends_on "ctags" => :optional
+  depends_on "universal-ctags/universal-ctags/universal-ctags" => :optional
```

Comment:
@MikeMcQuaid The [universal-ctags tap](https://github.com/universal-ctags/homebrew-universal-ctags) mentions,

> When Universal Ctags has a tagged release, this formula should be moved to the official Homebrew repository. This repo should then be deleted.

Indeed, [universal ctags](https://github.com/universal-ctags/ctags) still has no tagged release. I'll check with the maintainers on the status of that. Is a tagged release a prerequisite for submitting to homebrew/core?